frontend/packages/rhoas-plugin/console-extensions.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "type": "console.flag/model",
+    "properties": {
+      "flag": "RHOAS_FLAG",
+      "model": {
+        "group": "rhoas.redhat.com",
+        "version": "v1alpha1",
+        "kind": "CloudServiceAccountRequest"
+      }
+    }
+  },
+  {
+    "type": "dev-console.add/action",
+    "flags": {
+      "required": ["RHOAS_FLAG"]
+    },
+    "properties": {
+      "id": "rhoasAddAction",
+      "href": "/catalog?catalogType=managedservices",
+      "label": "%rhoas-plugin~Managed Services%",
+      "description": "%rhoas-plugin~Reduce operational complexity and focus on building and scaling applications that add more value.%",
+      "icon": { "$codeRef": "constants.cloudServicesIcon" }
```

Comment:
You need to pass `accessReview` properties to check if the user has create access to this resource.
Like we did for other add cards https://github.com/openshift/console/blob/master/frontend/packages/dev-console/console-extensions.json#L10-L16

Reply:
Thank you so much @vikram-raj! Applied this suggestion and added a new item to watch out in contribution guide